### PR TITLE
Case 21165: tangent attribute

### PR DESCRIPTION
**Pull request by raveenajain**
https://highfidelity.manuscript.com/f/cases/21165/gltf-tangents
https://github.com/highfidelity/hifi_tests/pull/356

### CODE REVIEW

**Comment by raveenajain** [comment]
Test Plan: 

1 - Run interface. Enter a domain with minimal external influences.
2 - Go to Edit -> Running Scripts -> From URL
3 - Output of running all of the scripts below should look like https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/NormalTangentMirrorTest

Scripts: 
https://raw.githubusercontent.com/raveenajain/hifi_tests/update_for_tangent/tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/embedded/metalRoughSpheres/test.js
https://raw.githubusercontent.com/raveenajain/hifi_tests/update_for_tangent/tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/metalRoughSpheres/test.js
https://raw.githubusercontent.com/raveenajain/hifi_tests/update_for_tangent/tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/metalRoughSpheres/test.js

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17262/HighFidelity-Beta-PR14914-a967d7a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17262/HighFidelity-Beta-PR14914-a967d7a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17262/HighFidelity-Beta-Interface-PR14914-a967d7a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17262/HighFidelity-Beta-PR14914-a967d7a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17262/HighFidelity-Beta-Interface-PR14914-a967d7a.exe

File: libraries/fbx/src/GLTFSerializer.cpp [hunk 1]
```diff
@@ -892,7 +892,24 @@ bool GLTFSerializer::buildGeometry(HFMModel& hfmModel, const QUrl& url) {
                         for (int n = 0; n < colors.size() - 3; n += stride) {
                             mesh.colors.push_back(glm::vec3(colors[n], colors[n + 1], colors[n + 2]));
                         }
-                    } else if (key == "TEXCOORD_0") {
+                    } else if (key == "TANGENT") {
```
Comment (samcake) [inline libraries/fbx/src/GLTFSerializer.cpp]: just a typo problem with the tab alignment

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17288/HighFidelity-Beta-PR14914-01def37.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17288/HighFidelity-Beta-PR14914-01def37.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17288/HighFidelity-Beta-Interface-PR14914-01def37.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17288/HighFidelity-Beta-PR14914-01def37.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17288/HighFidelity-Beta-Interface-PR14914-01def37.exe

**Pull request merged**